src/netaddress.h

```diff
@@ -12,6 +12,8 @@
 #include <attributes.h>
 #include <compat.h>
 #include <prevector.h>
+#include <crypto/siphash.h>
```

Comment:
nit: The latest rebasing destroyed alphabetic order of headers :)

Reply:
Fixed, someday we may consider to have an automated clang-format enforcement in CI...